lib/crowbar/init/application.rb

```diff
@@ -315,6 +315,9 @@ class Application < Sinatra::Base
       # api_version "2.0"
       post "/api/upgrade/new" do
         api_constraint(2.0)
+        upgrade_status = ::Crowbar::UpgradeStatus.new(logger)
+        upgrade_status.start_step
```

Comment:
It would be something like "reject API request if current_step is not as expected", right?